### DISCUSSION TITLE
oo2core_win64 doesn't exist

### DISCUSSION THREAD

**Discussion by lucasluc44** [gildor2/UEViewer#293] [Q&A] [answered]
hello 
I have a problem when i'm trying to build umodel in 64 bits
he say that oo2core_win64.lib doesn't exist after that he create a umodel.exe and when i'm trying to extract a pak file he say  oo2core_5_win64.dll not found.
could you please help me to solve this problem ?

(btw i'm french sorry if there spelling error)
![error1](https://user-images.githubusercontent.com/65608620/210565959-4d669f90-f6d6-45bd-b8b3-84f0e064b147.png)
![error2](https://user-images.githubusercontent.com/65608620/210565961-b9817e3d-716c-4c8e-b8b9-f8be201b4ba7.png)
![error2bis](https://user-images.githubusercontent.com/65608620/210565964-e81428b2-0fd4-45fa-bdd3-f91603a77c43.png)

**Comment by gildor2** [comment, marked as answer]
The lib/oodle directory has file README.md, read it.

**Reply by lucasluc44** [reply]
okay thanks
